app/scripts/components/common/cookie-consent/index.scss

```diff
@@ -0,0 +1,8 @@
+.modal {
+    z-index: 999;
+    position: fixed;
+    max-width: 725px;
+    border-radius: 35px;
```

Comment:
Could we use some of the radius classes from USWDS? https://designsystem.digital.gov/utilities/border/#utility-radius